### DISCUSSION TITLE
Feature Request - Partner Directory View within the Control Panel

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/console#78] [Plugin Store]
### Description
A bit of an “out there request”... I think it would be neat to have a view within the control panel to show craft partners (pulled from the partner network). Personally I think this could a great addition as it would provide access to this information in for example, the context of building out a new craft site.

I can imagine instances where a developer or professional services supplier could be building out a new site and decide to outsource some of the requirements. Having the partner directory listing within the control panel (and potentially listing partners based on the locale set for the control panel as default) could be super cool.

I would bet there are plenty of people who are completely unaware of the partner network or the fact that there are a growing number of small businesses that offer specialist craftcms expertise.

**Comment by brandonkelly** [comment]
Including a full blown partner listing in the CP seems a little weird, however we have thought about visually distinguishing developers’ avatars and profiles within the Plugin Store if they are also a partner, to inspire confidence in their plugins. Maybe around the same time we could also add a “Developers” listing to the Plugin Store, and have a way to filter by partner-developers.

**Comment by brandonkelly** [comment]
Also, for all the times that we get support from clients that are no longer in touch with their developer, it would probably make sense to link out to https://craftcms.com/partners, perhaps from the Craft Support widget or something.
